docs/design/federation-phase-1.md

```diff
@@ -0,0 +1,386 @@
+# Ubernetes Design Spec (phase one)
+
+**Huawei PaaS Team**
```

Comment:
@roberthbailey Both author and intended implementor.  The reasons for collapsing into a single PR were purely pragmatic.  The Huawei are free to respond to comments, and have in some cases.  We can split this doc into a separate PR if the author attribution issue of real concern to anyone?